### DISCUSSION TITLE
Inventory Settings

### DISCUSSION THREAD

**Discussion by gattarmax** [isid0re/SoloLeveling#281] [General]
Config.Inventory[0] = [0,0,1,1,1,1,1,1,1,1];
Config.Inventory[1] = [0,0,1,1,1,1,1,1,1,1];
Config.Inventory[2] = [0,0,1,1,1,1,1,1,1,1];
Config.Inventory[3] = [0,0,1,1,1,1,1,1,1,1];

Inventory Settings
Why would I still move my belongings??
Thank

Which file should I set ?

**Comment by isid0re** [comment]
Inventory lock doesnt work at this time.

**Reply by gattarmax** [reply]
Thank
